docs/components/content/examples/FormExampleValibot.vue

```diff
@@ -1,13 +1,13 @@
 <script setup lang="ts">
-import { string, objectAsync, email, minLength, type Input } from 'valibot'
+import * as v from 'valibot'
 import type { FormSubmitEvent } from '#ui/types'
 
-const schema = objectAsync({
-  email: string([email('Invalid email')]),
-  password: string([minLength(8, 'Must be at least 8 characters')])
+const schema = v.object({
```

Comment:
Shouldn't we wrap this into`v.safeParser` to make the example compatible with valibot's global config?

Reply:
I think most people will use it without, but it's probably a good idea to document the use of `v.safeParser` somehow.

Reply:
If that's the scenario that will always work, It would be great to include it in the default example since it'll be most people's starting point. What do you think?

Reply:
Done